src/Microsoft.AspNet.Mvc.TagHelpers/ValidationMessageTagHelper.cs

```diff
@@ -8,10 +8,10 @@
 namespace Microsoft.AspNet.Mvc.TagHelpers
 {
     /// <summary>
-    /// <see cref="ITagHelper"/> implementation targeting &lt;span&gt; elements with an <c>asp-validation-for</c>
+    /// <see cref="ITagHelper"/> implementation targeting any HTML element with an <c>asp-validation-for</c>
     /// attribute.
     /// </summary>
-    [TargetElement("span")]
+    [TargetElement(Attributes = ValidationForAttributeName)]
```

Comment:
this change in behaviour should be separate.  discuss w/ @DamianEdwards as to whether we need to make this change.